superset-frontend/src/components/DeleteModal/DeleteModal.test.tsx

```diff
@@ -0,0 +1,95 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { supersetTheme, ThemeProvider } from '@superset-ui/core';
+import '@testing-library/jest-dom';
+import { render, screen, configure } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import DeleteModal from '.';
+
+configure({ testIdAttribute: 'data-test' });
```

Comment:
Please just change [`spec/helpers/setup.ts`](https://github.com/apache/superset/blob/86807e40b778495d3f5155e043fd6f61d787f17e/superset-frontend/spec/helpers/setup.ts#L23) as this config should be global.

Reply:
@ktmud I don't think it's a good idea to change there. This can impact other tests and that is not the intention.

I believe that this global configuration should be the scope of another PR.

Reply:
It's very easy to do a search & replace:

<img src="https://user-images.githubusercontent.com/335541/108791300-9647c580-7533-11eb-981f-2e823568a0ec.png" width="420">

We introduced RTL pretty recently and there's only one component that is currently using `data-test-id` so I wouldn't worry about touching the global.

It's OK to change the global in another PR, but if you do, please remember to clean up the `configure` here.

Reply:
I really prefer to do this global setup and clear everything in another PR. It makes sense to make PRs as atomic as possible.
If you want, you can open an issue and sign it for me, tomorrow I'll solve it

Reply:
No need: https://github.com/apache/superset/pull/13292

Reply:
@yardz it's late in Brazil, get good rest. 🙂

Reply:
@yardz Can you clean this up now that #13292 is merged?

Reply:
@ktmud done!